src/languages/r.coffee

```diff
@@ -0,0 +1,35 @@
+# Get Atom defaults
+scope = ['text.html']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 1)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
+module.exports = {
+
+  name: "R"
+  namespace: "r"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "R"
+]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "r"
+    "R"
+  ]
+
+  options:
+    indent_size:
```

Comment:
This looks correct. Maybe try building the docs again and see if the option shows up in the docs. If so, then also try reloading Atom to see if it shows up in the settings now.